tests/template_tests/test_nodelist.py

```diff
@@ -32,6 +33,21 @@ def test_ifchanged(self):
         self.assertEqual(len(vars), 1)
 
 
+class TextNodeReprTest(TestCase):
+
+    def test_textnode_repr(self):
+        engine = Engine()
```

Comment:
I'll keep it as `TextNodeTest`.  I don't understand the point of the setUpClass: why tie the tests together with a class attribute.  Creating an Engine isn't an expensive operation, is it?